main/modes/platformer/mode_platformer.c

```diff
@@ -367,7 +336,7 @@ void drawPlatformerTitleScreen(display_t *d, font_t *font, gameData_t *gameData)
 
     if (gameData->frameCount < 10)
     {
-        drawText(d, font, c555, "Press A or B to start", 20, 128);
+        drawText(d, font, c555, "- Press START button -", 20, 128);
```

Comment:
If you want to center text, don't use fixed offsets. Use `textWidth()` to find the width before drawing, then find the correct offset. For example:
```C
// Center and draw the text
int16_t tWidth = textWidth(&credits->radiostars, creditNames[idx]);
drawText(credits->disp, &credits->radiostars, creditColors[idx], creditNames[idx],
            (credits->disp->w - tWidth) / 2, (yPos + credits->yOffset));
```

Reply:
I'll keep this in mind. Will probably fix it later.

Reply:
As long as you're aware